Demo/obj/Release/netcoreapp3.0/Demo.csproj.FileListAbsolute.txt

```diff
@@ -0,0 +1,44 @@
+C:\Users\Yas\source\repos\Testbed-GatewayServer\Demo\bin\Release\netcoreapp3.0\appsettings.Development.json
```

Comment:
Why do we need your local dirs?

Reply:
And I believe we don't need any obj files below

Reply:
This code has been removed